pkgs/development/libraries/fltk/default.nix

```diff
@@ -39,10 +54,18 @@ composableDerivation.composableDerivation {} rec {
     localpngSupport = false;
     sharedSupport = true;
     threadsSupport = true;
-  };
+  } // cfg;
 
   meta = {
-    description = "a C++ cross platform lightweight gui library binding";
+    description = "A C++ cross-platform lightweight GUI library";
     homepage = http://www.fltk.org;
+    platforms = stdenv.lib.platforms.linux;
+    license = stdenv.lib.licenses.gpl2;
   };
+
+  # doesn't build without it. Why?
+  patches = [
```

Comment:
Tried both. Both failed. And now the previous expression doesn't build anymore. Cool...